ansible/roles/robogardener/meta/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+galaxy_info:
+  author: Rémy Greinhofer
+  description: Deploy the Robot Gardener software
```

Comment:
This is actually only code of The System Controller project.  Can we change the references to The Robot Gardener to The System Controller through out this code?